<?xml version="1.0" encoding="UTF-8"?>
<testrun name="ExperimentDAOTest" project="GemsJaxServer" tests="1" started="0" failures="0" errors="2" ignored="0">
  <testsuite name="test.persistence.ExperimentDAOTest" incomplete="true">
    <error>java.lang.ExceptionInInitializerError
	at org.gemsjax.server.persistence.dao.ExperimentDAO.&lt;init&gt;(ExperimentDAO.java:46)
	at test.persistence.ExperimentDAOTest.classSetup(ExperimentDAOTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:616)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:27)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:46)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
Caused by: org.hibernate.InvalidMappingException: Unable to read XML
	at org.hibernate.util.xml.MappingReader.readMappingDocument(MappingReader.java:101)
	at org.hibernate.cfg.Configuration.add(Configuration.java:513)
	at org.hibernate.cfg.Configuration.add(Configuration.java:509)
	at org.hibernate.cfg.Configuration.add(Configuration.java:716)
	at org.hibernate.cfg.Configuration.addResource(Configuration.java:801)
	at org.hibernate.cfg.Configuration.parseMappingElement(Configuration.java:2344)
	at org.hibernate.cfg.Configuration.parseSessionFactory(Configuration.java:2310)
	at org.hibernate.cfg.Configuration.doConfigure(Configuration.java:2290)
	at org.hibernate.cfg.Configuration.doConfigure(Configuration.java:2243)
	at org.hibernate.cfg.Configuration.configure(Configuration.java:2158)
	at org.hibernate.cfg.Configuration.configure(Configuration.java:2137)
	at org.gemsjax.server.persistence.HibernateUtil.&lt;clinit&gt;(HibernateUtil.java:16)
	... 18 more
Caused by: org.dom4j.DocumentException: Error on line 20 of document  : Open quote is expected for attribute "{1}" associated with an  element type  "table". Nested exception: Open quote is expected for attribute "{1}" associated with an  element type  "table".
	at org.dom4j.io.SAXReader.read(SAXReader.java:482)
	at org.hibernate.util.xml.MappingReader.readMappingDocument(MappingReader.java:75)
	... 29 more

java.lang.NullPointerException
	at test.persistence.ExperimentDAOTest.classSetDown(ExperimentDAOTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:616)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:37)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:46)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</error>
    <testcase name="experimentTests" classname="test.persistence.ExperimentDAOTest" incomplete="true"/>
  </testsuite>
</testrun>
